src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,9 @@
+export default function ReviewCard({ title, category, reviewImgUrl }) {
+  return (
+    <article>
+      <h3>Title: {title}</h3>
+      <p>Category: {category}</p>
+      <img src={reviewImgUrl} alt={title} className="review-img"></img>
+    </article>
```

Comment:
It might be a good idea to display the date for reviews as this will come in handy when you are sorting them 👍